lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1103,7 +1103,7 @@ def test_static_badlink_rewrite(self):
         result_fragment = module.render(STUDENT_VIEW)
 
         self.assertIn(
-            '/c4x/{org}/{course}/asset/_file.jpg'.format(
+            '/c4x/{org}/{course}/asset/file.jpg'.format(
```

Comment:
The real question is: for courses that upload assets organized in subdirectories, do they use the real `subdir/file.png` path or edit their courseware to reference the flattened `subdir_file.png` path?

Both will work correctly -- rewritten to subdir_file.png during canonicalization -- with this PR, but I'm curious for pedagogical value.